packages/l10n/country-information.js

```diff
@@ -3,6 +3,25 @@ import createL10NInjector from './create-l10n-injector';
 import getSupportedLanguage from './utils/get-supported-language';
 import extractLanguageFromLocale from './utils/extract-language-from-locale';
 
+const getImportChunk = lang => {
+  let importChunk;
+  switch (lang) {
+    case 'en':
+      importChunk = import(/* webpackChunkName: "country-data-en" */ './data/countries/en.json');
```

Comment:
Same nitpick as the other PR 🤪

PS: you can also `return import(...)` directly instead of assigning a local var